jogo/src/entity/hud/modal_content.ts

```diff
@@ -0,0 +1,14 @@
+import { template } from 'handlebars'
+import { Store } from '../../infra/store'
+import { Component } from './component'
+
+export class ModalContent extends Component<void> {
+  constructor(store: Store<void>, private templatePath: string) {
+    super(store)
+  }
+
+  render(): string {
+    const template = require('./template/back_button.hbs')
+    return template({})
+  }
+}
```

Comment:
o mesmo sobre o comentário do newline: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline